### PR TITLE
feat: function to load in datasets.

### PR DESCRIPTION
**Pull request by kkahadze**

### CODE REVIEW

**Comment by trigaten** [comment]
Rename `load.py` to smtg `load_hf_data.py`

File: src/prompt_systematic_review/load.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    :param name: Defines the name of the dataset configuration
+    :type name: str
+    :param split: Which split of the data to load. If None, will return a dict with all splits (typically datasets.Split.TRAIN and datasets.Split.TEST).
+    :type split: str or Split
+    :return: The loaded dataset.
+    :rtype: Dataset or DatasetDict
+    """
+
+    try:
+        if name:
```
Comment (trigaten) [inline src/prompt_systematic_review/load.py:19]: Double if is interesting :)

File: src/prompt_systematic_review/load.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+from datasets import load_dataset
+
+
+def load_hf_dataset(dataset_name, name=None, split=None):
```
Comment (trigaten) [inline src/prompt_systematic_review/load.py:4]: Please add type hints

File: src/prompt_systematic_review/load.py (hunk 1 above)
Comment (kkahadze) [reply src/prompt_systematic_review/load.py:19]: I've gotten rusty : ). Made it all one line.

File: src/prompt_systematic_review/load.py (hunk 2 above)
Comment (kkahadze) [reply src/prompt_systematic_review/load.py:4]: Done

**Pull request merged**